src/transaction/log_impl.h

```diff
@@ -652,10 +654,16 @@ struct log_global
 #endif // SERVER_MODE = !SA_MODE
   // *INDENT-ON*
 
+  std::mutex ps_lsa_mutex;
+  std::condition_variable ps_lsa_cv;
+  LOG_LSA max_ps_flushed_lsa;
+
   // *INDENT-OFF*
```

Comment:
All namespaced variables need to have indent disabled. Add m_ prefix to the names of member variables.
```suggestion
  // *INDENT-OFF*
  std::mutex m_ps_lsa_mutex;
  std::condition_variable m_ps_lsa_cv;
  LOG_LSA m_max_ps_flushed_lsa;

```